### PR TITLE
Back to basics: Technical DEBT

### PR DESCRIPTION
**Pull request by wpschaub**
A DRAFT Pr with a few thoughts on technical debt. What is missing?

### CODE REVIEW

**Review by MerlinDarkstar — APPROVED** [review]

File: content/back-to-basics-dealing-with-technical-debt.md
Comment (AndreKaminski) [inline content/back-to-basics-dealing-with-technical-debt.md:1]: "When I glance at our [Azure DevOps](https://learn.microsoft.com/en-us/azure/devops/user-guide/what-is-azure-devops?view=azure-devops) technical debt backlog, it is like revisiting the unforgettable scene from Jurassic Park, where Ian Malcolm walks up to a pile of dinosaur s**t." I would rephrase this, it is not very inspirational, and tech debt will always exist, it needs to be part of our daily life

Comment (wpschaub) [reply content/back-to-basics-dealing-with-technical-debt.md:1]: @AndreKaminski do you want it rephrased or removed? If rephrased, would this sound better, whereby I battle to make technical debt inspirational?

"When I first looked at our technical debt, consolidated on one of our Azure DevOps backlogs for tracking, I felt like entering a scene from Jurassic Park, where Ian Malcolm bravely walks up to a pile of challenges, eager to solve them."
